src/select/autosize-input.js

```diff
@@ -76,7 +76,12 @@ export default class AutosizeInput extends React.Component<
       <React.Fragment>
         <Input {...componentInputProps} ref={inputRef} {...inputProps} />
         {/* a hidden helper element to calculate the size of the input */}
-        <StyledInputSizer $size={this.props.$size} ref={this.sizerRef}>
+        <StyledInputSizer
+          $size={this.props.$size}
+          ref={this.sizerRef}
+          // $FlowFixMe checking for $style before use
+          $style={inputProps.$style ? inputProps.$style : null}
```

Comment:
does undefined rather than null pass flow check?

Reply:
`inputProps`, which is returned by a `getOverrides` call, is typed as `{}` so `inputProps.$style` isn't allowed. not sure if we can alter the return type for props on `getOverrides`?

Reply:
i think it's safe to change that return type to {$style?: any} but may be better to save for another pr